CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 ###############################################################################
 # CMake build rules for Micro CDR
 ###############################################################################
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
```

Comment:
CI is not updated to have this new version. Linux jobs are failing due to it. 
http://jenkins.eprosima.com:8080/view/Micro%20XRCE/job/Micro-CDR%20Nightly%20Master%20Linux/1232/console

```
[Micro-CDR Nightly Master Linux] $ /home/jenkins/tools/hudson.plugins.cmake.CmakeTool/3.10.0/bin/cmake --version
cmake version 3.10.0
...
CMake Error at CMakeLists.txt:18 (cmake_minimum_required):
  CMake 3.15 or higher is required.  You are running version 3.10.0


-- Configuring incomplete, errors occurred!
```

Reply:
@richiware